src/content/release-notes.mdx

```diff
@@ -5,6 +5,17 @@ sidebar_label: Release notes
 id: release-notes
 ---
 
+## 1.28.2
+
+14 January 2025
+
+This version is compatible with Kubernetes versions 1.27 to 1.30 \
+Okteto Chart release 1.28 is designed to work with [Okteto CLI 3.3.x](https://github.com/okteto/okteto/releases/tag/3.3.0)
```

Comment:
I think in this link we use the latest patch of the CLI. In this case it would be the same than the one in the bug fixes item.